docs-search/src/Docs/Search/BrowserEngine.purs

```diff
@@ -2,21 +2,22 @@
 module Docs.Search.BrowserEngine where
 
 import Docs.Search.Config as Config
-import Docs.Search.PackageIndex (queryPackageIndex)
 import Docs.Search.Engine (Engine, EngineState, Index)
-import Docs.Search.SearchResult (SearchResult)
+import Docs.Search.ModuleIndex as ModuleIndex
+import Docs.Search.PackageIndex (queryPackageIndex)
+import Docs.Search.SearchResult (SearchResult, searchResultCodec)
```

Comment:
The import style throughout the whole Spago codebase is to:
- import everything qualified
- ..except that types can be imported unqualified

So this would become:
```suggestion
import Docs.Search.ModuleIndex as ModuleIndex
import Docs.Search.PackageIndex as PackageIndex 
import Docs.Search.SearchResult as SearchResult
import Docs.Search.SearchResult (SearchResult)
```

Reply:
Sure, I'll make the changes to use qualified names for the imports that I touch in this PR (and future ones).